code/go/internal/validator/folder.go

```diff
@@ -0,0 +1,217 @@
+package validator
+
+import (
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"os"
+	"path"
+	"path/filepath"
+	"regexp"
+
+	"github.com/pkg/errors"
+	"gopkg.in/yaml.v3"
+)
+
+const ItemTypeFile = "file"
+const ItemTypeFolder = "folder"
```

Comment:
should theses consts be exposed?

Reply:
Fixed in 11f07afc3013fc66a81701d2e64fde29de87900a.